project/Dependencies.scala

```diff
@@ -277,6 +277,8 @@ object Dependencies {
   val Kinesis = Seq(
     libraryDependencies ++= Seq(
         "com.amazonaws" % "aws-java-sdk-kinesis" % AwsSdkVersion, // ApacheV2
+        "software.amazon.awssdk" % "kinesis" % AwsSdk2Version, // ApacheV2
```

Comment:
Having AWS SDK 1 and 2 on the dependency list isn't great. Before merging this we should fix #1347